pext/qml/InstallThemeFromURLDialog.qml

```diff
@@ -60,6 +60,9 @@ Dialog {
 
         xmlhttp.open("GET", textfield.text, true);
         xmlhttp.send();
+
+        destroy();
```

Comment:
Please put this destroy inside the onreadystatechange after installRequest is called

Reply:
Commited! :D